.github/check-api-version.py

```diff
@@ -0,0 +1,12 @@
+import requests
+import re
+import sys
+import pkg_resources  # part of setuptools
+
+package_version = pkg_resources.require("openfisca-france")[0].version
+
+response_API = requests.get('https://fr.openfisca.org/legislation/swagger')
+data = response_API.text
+api_package_version = re.search('countryPackageVersion":"(.*)","entities', data).group(1)
```

Comment:
Parsing the rendered HTML is convoluted and fragile. Use the dedicated `info.version` field in `https://fr.openfisca.org/api/latest/spec` instead or, if you want to minimise the weight, use the `Country-Package-Version` header field in `https://fr.openfisca.org/api/latest/` 🙂